did-docs.html

```diff
@@ -114,6 +112,23 @@ <h2>CRDTs</h2>
 
 </section>
 
+<section>
+    <h2><code>initial-state</code></h2>
+    <p>
+        Sometimes it is useful (or even required) to communicate a doc-capable peer DID value (one with <code>numalgo</code> != 0) and its genesis or evolved DID doc as a single combined value, instead of sharing the DID as one piece of data, and its DID doc and deltas as another. This situation commonly occurs in connection protocols that are built on <a target="_blank" href="https://identity.foundation/didcomm-messaging/docs/spec/">DIDComm messaging</a>, for example: <var>A</var> wants to share with <var>B</var> the new pairwise DID they've allocated, plus the DID's full DID doc, and DIDComm requires all information about the sender to be passed in a single <code>from</code> field.
```

Comment:
Should we use the ssi notation instead of A and B in the example?

Reply:
Good improvement! Will update.

Reply:
Fixed.